lib/ansible/plugins/inventory/vmware_vm_inventory.py

```diff
@@ -47,11 +53,21 @@
             default: 443
             env:
               - name: VMWARE_PORT
+        search_folder:
+            description: Inventory path for the root folder of the VM search. Requires `datacenter` to be set.
+            required: False
+            type: str
+            version_added: "2.8"
         validate_certs:
             description:
             - Allows connection when SSL certificates are not valid. Set to C(false) when certificates are not trusted.
             default: True
             type: boolean
+        with_folders:
+            description: Create groups based on the folder hierarchy of the VCenter Inventory. Requires  `datacenter` to be set.
```

Comment:
you should not need to do this as a specific feature, using the 'constructed/compose' base class will allow users to do this and a lot more in a dynamic fashion w/o having to hard code it.